### ISSUE TITLE
HelloHacktoberfest - Angular

### ISSUE THREAD

**Issue by ajjayymahato** [ajjayymahato/HelloHacktoberfest#9]

**Comment by Rhythm-08** [comment]
What are we supposed to do in this issue?
Just need to write hellohacktoberfest, then please @ajjayymahato assign it to me

**Comment by ajjayymahato** [comment]
Yes you need to print Hello Hacktoberfest in Angular, you can also use any creativity to make it unique. Also do give stars to the repo, and check out other ToDoApp Repo as well.
